docker/Dockerfile.test.sqlite.clang7.bionic

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+  && apt-get install -y software-properties-common wget \
+  && wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - \
+  && add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" \
+  && apt-get update \
+  && apt-get install -y \
```

Comment:
We can add `--no-install-recommends` switch to `apt-get` which will eliminate recommended packages and reduce the image size. Please add it to the other docker file too.